com.unity.render-pipelines.core/Runtime/Utilities/ResourceReloader.cs

```diff
@@ -96,6 +96,15 @@ public static bool ReloadAllNullIn(System.Object container, string basePath)
                         }
                     }
                 }
+                else if (fieldInfo.FieldType.IsArray)
```

Comment:
If I get correctly, this will allow to search ReloadGroup in our RendererFeature array?

Reply:
Can't it be done by the Reload attribute? Like in:
> [Reload(new[]
            {
                "Textures/FilmGrain/Thin01.png",
                "Textures/FilmGrain/Thin02.png",
                "Textures/FilmGrain/Medium01.png",
                "Textures/FilmGrain/Medium02.png",
                "Textures/FilmGrain/Medium03.png",
                "Textures/FilmGrain/Medium04.png",
                "Textures/FilmGrain/Medium05.png",
                "Textures/FilmGrain/Medium06.png",
                "Textures/FilmGrain/Large01.png",
                "Textures/FilmGrain/Large02.png"
            })]
            public Texture2D[] filmGrainTex;